### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by rueyjye**
Fix readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,52 @@
...
-* Tests run in parallel
-* Re-run function
-* Codes of test suite are of the form YAML
-* Multiple browser support (Chrome)
-* Some Test Categories support
-  * E2E Tests including JavaScript Error Check
-  * LINK Status Tests in web page
-* Test Report by [bucky-management](https://github.com/lifull-dev/bucky-management)
+* Run tests in parallel
+* Re-run tests whice failed last time
```
Comment (hikimochi) [inline README.md]: whice?

File: README.md [hunk 2]
```diff
@@ -2,20 +2,52 @@
...
+* Support test code in YAML
+* Multiple browser supported (Currently only Chrome is supported)
+* Customizable test categories
+  * [Default] E2E: E2E tests including JavaScript error check
```
Comment (hikimochi) [inline README.md]: please add note 'end to end'

File: README.md [hunk 3]
```diff
@@ -2,20 +2,52 @@
...
+  * [Default] Linkstatus: http respond check in web page
```
Comment (hikimochi) [inline README.md]: I think http status code check is better.

File: README.md [hunk 4]
```diff
@@ -2,20 +2,52 @@
...
+# Make pageobject(.rb) and part(.yml) in device directory
+## PC
+bucky make page {page_name} --service {your_service_name} --device pc
+## Smart phone
+bucky make page {page_name} --service {your_service_name} --device sp
+## Tablet
+bucky make page {page_name} --service {your_service_name} --device tablet
+
+# Write your test code in following directory:
+# services/{your_service_name}/{devce}/scenarios/e2e/
```
Comment (hikimochi) [inline README.md]: devce

File: README.md [hunk 5]
```diff
@@ -2,20 +2,52 @@
...
+# services/{your_service_name}/{devce}/scenarios/linkstatus/
```
Comment (hikimochi) [inline README.md]: devce

File: README.md [hunk 6]
```diff
@@ -26,72 +58,53 @@ export BUCKY_DB_NAME="{your database name}"
...
-bucky run -t e2e
-bucky run -t linkstatus
-# Filter test condition
+# Condition filter using option
 bucky run --test_category e2e --device sp --priority high
 bucky run --test_category e2e --case login
 bucky run --test_category e2e -D tablet --priority high
 bucky run --test_category e2e -D pc --label foo,bar,baz --priority high
-# For debug (-d,--debug: not insert test result)
+# Run test in debug mod (It won't insert test result into DB)
```
Comment (hikimochi) [inline README.md]: mod

File: README.md (hunk 1 above)
Comment (rueyjye) [reply README.md]: typo which

File: README.md (hunk 2 above)
Comment (rueyjye) [reply README.md]: thanks

File: README.md (hunk 3 above)
Comment (rueyjye) [reply README.md]: agree!

File: README.md (hunk 4 above)
Comment (rueyjye) [reply README.md]: typo device

File: README.md (hunk 5 above)
Comment (rueyjye) [reply README.md]: typo device

File: README.md (hunk 6 above)
Comment (rueyjye) [reply README.md]: typo mode

**Comment by hikimochi** [comment]
LGTM

**Pull request merged**